src/Canvas.tsx

```diff
@@ -431,6 +431,7 @@ const InternalCanvas: FC<CanvasProps & { ref?: Ref<CanvasRef> }> = forwardRef(
                   key={e.id}
                   element={element}
                   disabled={disabled}
+                  children={element.props.children}
```

Comment:
I don't think you can call this `children` since that is a reserved property name by react. I think I call it something else for `Node` - can you copy that name?

Reply:
It's the exact same as the Node code, it's where I copied it from: https://github.com/reaviz/reaflow/pull/136/files/788c20ce9efc5c189dba8f8902ee12db612ada5e#R416

Reply:
I'm sorry - I misread this code I thought it was models not react nodes. LGTM

Reply:
All good, thank you for merging :)